Kickstarter-iOS/Views/Storyboards/Settings.storyboard

```diff
@@ -205,17 +205,17 @@
                                             <view hidden="YES" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="Cdr-bL-P2W" userLabel="Message Label View">
                                                 <rect key="frame" x="0.0" y="88" width="375" height="24"/>
                                                 <subviews>
-                                                    <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" alignment="top" translatesAutoresizingMaskIntoConstraints="NO" id="IjC-cR-RB2">
+                                                    <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" translatesAutoresizingMaskIntoConstraints="NO" id="IjC-cR-RB2">
                                                         <rect key="frame" x="12" y="6" width="351" height="0.0"/>
                                                         <subviews>
-                                                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="255" verticalHuggingPriority="255" text="Label" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="lz0-3G-6S5">
-                                                                <rect key="frame" x="0.0" y="0.0" width="42" height="0.0"/>
+                                                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" text="Label" lineBreakMode="wordWrap" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="lz0-3G-6S5">
```

Comment:
I think it would also be better if we cans set these in code? It's just overall simpler to make changes to that later on and easier to find (wrap in a styles)....I think this is something we should all talk about on our team to see if this is the pattern we want to follow or not

Reply:
This is also not needed and is being set [via a styles variable](https://github.com/kickstarter/ios-oss/blob/2278546279f5360a350f095adc75a20b39f0419e/Kickstarter-iOS/Views/Controllers/ChangeEmailViewController.swift#L85). Will also remove!